src/domain/event/eventBasicInfo/EventCategorisation.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+
+import {
+  EventFieldsFragment,
+  LocalisedFieldsFragment,
+} from '../../../generated/graphql';
+import useLocale from '../../../hooks/useLocale';
+import { useTranslation } from '../../../i18n';
+import getLocalizedString from '../../../utils/getLocalisedString';
+import { getEventFields, getRealKeywords } from '../utils';
+import styles from './eventBasicInfo.module.scss';
+
+export const categorisationSectionTestId = 'categorisation-section-testid';
+
+const EventCategorisation: React.FC<{
+  event: EventFieldsFragment | null | undefined;
+}> = ({ event }) => {
+  const locale = useLocale();
+  const realKeywords = getRealKeywords({ event });
+  const { t } = useTranslation();
+  const { categories, activities, audience } = getEventFields(event, locale);
+
+  const arrayToText = (
+    items: { name?: LocalisedFieldsFragment | null }[] | undefined
+  ) => {
+    return (
+      items
+        ?.map((item) => getLocalizedString(item.name || {}, locale))
+        .filter(Boolean)
+        .sort()
+        .join(', ') || '-'
+    );
+  };
```

Comment:
Could this be a domain util or something?

Reply:
Hmm I just used this same way like it was used in admin project. It could be domain util function of course :)

Reply:
Moved to utils